ToDoApp/ToDoApp/CreateTaskView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CreateTaskView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-01.
+//
+
+import Foundation
+import SwiftUI
+
+struct CreateTaskView: View {
+    @StateObject private var createTaskViewModel = CreateTaskViewModel()
+   var body: some View {
+       VStack(spacing: 15) {
+           NavigationStack{
+               VStack(alignment: .leading, spacing: 15){
+                   Text("Task Title")
+                       .multilineTextAlignment(.leading)
+                   TextField("Insert Title", text: $createTaskViewModel.titleInput)
+                       .padding(.leading, 10.0)
```

Comment:
```suggestion
                       .padding(.horizontal, 10.0)
```
you should use horizontal padding instead of leading since when there would be a lot of text there should be trailing padding as well

Reply:
This has been fixed.